### DISCUSSION TITLE
Am I running this correctly?

### DISCUSSION THREAD

**Discussion by djthornton1212** [NickLiffen/ghas-enablement#60] [General]
Hello @NickLiffen I'm noticing that when I run the getRepos it's updating the repos.json as expected, but the contents look incomplete.

Example:
It starts as...
```
[
  {
    "login": "octodemo",
    "repos": [
      {
        "enableDependabot": "boolean",
        "enableDependabotUpdates": "boolean",
        "enableSecretScanning": "boolean",
        "enableCodeScanning": "boolean",
        "createIssue": "boolean",
        "repo": "org/repo"
      }
    ]
  }
]
```
Then I run `yarn run getRepos` yields... 
```
[
  {
    "login": "<org_1>",
    "repos": []
  },
  {
    "login": "<org_2>",
    "repos": []
  }
]
```

The logs look good too. It list all the repos, their languages, etc. It also says ...
```
ghas:inform Success created repos.json +1ms
Done in 8.63s
```

Am I missing something.

**Comment by NickLiffen** [comment]
@djthornton1212 👋 

Do you have access to all the organisations within this enterprise 👀 you are going to need access to every organisation. Being an enterprise admin I don't think cut's it, I think you need to be a direct admin on every org 👀

**Reply by djthornton1212** [reply]
That's correct, I am an organization owner for both:
![image](https://user-images.githubusercontent.com/99346713/162469509-6456ec2c-33b7-43c5-8029-19f1d2884fed.png)

And as I said in the post above I can see it list each and every repo in the organizations in the logs...

I think I see the problem after completely reading the logs:

> ghas:inform Found 0 repositories that met the valid criteria in the organisation <company>. Out of 96.

**Reply by djthornton1212** [reply]
Is the language option a list or should it be a single language? I'm guessing a filter for a single option. I tried adding multiple languages to filter on and it matched none. I removed it entirely and all repositories matched.

Is it expected that we should run 1 language at a time?

**Reply by NickLiffen** [reply]
> Is it expected that we should run 1 language at a time?

Okay, I see where you are going 🙇 So yes, it is only meant to run one language at a time. The reason why we do this is that the `codeql-analysis.yml` is different, per language. So you are only meant to run it with one language at a time. E.G start with `javascript`, then move to `python`, etc 👍 

This will allow you to enable the right `codeql-analysis.yml` across multiple files. 

If you don't want to enable CodeQL, just remove the language, make it empty, and you are good to go just with secret scanning and dependabot 👍

**Reply by djthornton1212** [reply]
Got it. Ok, I was thinking that it would apply the corresponding codeql-analysis.yml to the language found per repo in one go. 

Ok, I can make this work then. Thank you very much for the feedback.

**Reply by NickLiffen** [reply]
No problem 🙇 Please keep this feedback coming 🙇
